recipe/build.sh

```diff
@@ -8,10 +8,12 @@ unset SUBDIR
         --disable-doc \
         --enable-shared \
         --enable-static \
-        --extra-cflags="-Wall -g -m64 -pipe -O3 -march=x86-64 -fPIC `pkg-config --cflags zlib`" \
```

Comment:
FTR (in case anyone wonders) checked that `zlib` is correctly linked using the package and not the system copy by looking at the logs. So this is ok to drop now. :)